libs/login/gradlew.bat

```diff
@@ -29,9 +29,6 @@ if "%DIRNAME%" == "" set DIRNAME=.
 set APP_BASE_NAME=%~n0
```

Comment:
The changes in this file have been made by running `./gradlew wrapper --gradle-version 6.1.1 --distribution-type all` and should be ignored.